README.md

```diff
@@ -22,6 +22,10 @@
 OpenTTD is a transport simulation game based upon the popular game Transport Tycoon Deluxe, written by Chris Sawyer.
 It attempts to mimic the original game as closely as possible while extending it with new features.
 
+hen played, OpenTTD includes many ways of expanding and connecting many ports or cities that you may have built.
```

Comment:
Missing a `W`? 🙂 

This section is worded strangely, to me. I would expect it to mention transporting cargo, not just connecting cities and ports. (Why ports? And not industries?)

As an alternative, you might copy-and-paste the description (which full disclosure, I wrote) on the OpenTTD Steam store page: https://store.steampowered.com/app/1536610/OpenTTD/